staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/dynamic_serving_content.go

```diff
@@ -53,7 +53,7 @@ var _ ControllerRunner = &DynamicCertKeyPairContent{}
 // NewDynamicServingContentFromFiles returns a dynamic CertKeyContentProvider based on a cert and key filename
 func NewDynamicServingContentFromFiles(purpose, certFile, keyFile string) (*DynamicCertKeyPairContent, error) {
 	if len(certFile) == 0 || len(keyFile) == 0 {
-		return nil, fmt.Errorf("missing filename for serving cert")
+		return nil, fmt.Errorf("missing one of filename(--tls-cert-file\--tls-private-key-file) for serving cert")
```

Comment:
This is actually used from many locations, including SNI certificates for instance.  If you want an error message about these two flags, you'll want to move it into a validation function of the options themselves before you reach this point.

This is library code, which cannot be opinionated about flag names.

You can find me on kube slack (deads2k) for a re-review